lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -358,6 +358,8 @@ def git_info_for_course(self, cdir):
             info = [output_json['commit'],
                     output_json['date'],
                     output_json['author'], ]
+        except OSError as error:
+            log.warning(text_type(u"Error fetching git data: %s - %s"), text_type(cdir), text_type(error))
```

Comment:
I'm not sure if I'm belt-and-suspenders-ing this or what;
I kept iterating `text_type`/`unicode` calls until quality checks stopped complaining.

What's the preferred idiom here for unicode string handling at present?

Reply:
I think the first `text_type` here is unnecessary, but I also don't think it does anything bad, and I'm not going to hold up the PR over it.

Reply:
Sounds good. I'll revisit in my follow-up PR.